### PR TITLE
updated phpdoc for route callback argument

### PR DESCRIPTION
**Pull request by richard-ejem**

### CODE REVIEW

**Comment by dg** [comment]
Nette\Callback is obsolete, \Closure is enough

**Comment by richard-ejem** [comment]
fixed

**Comment by dg** [comment]
Can you change commit message to something like `Route: typo` or `Route: phpDoc typo`.

File: src/Application/Routers/Route.php [hunk 1]
```diff
@@ -106,7 +106,7 @@ class Route extends Nette\Object implements Application\IRouter
 
 	/**
 	 * @param  string  URL mask, e.g. '<presenter>/<action>/<id \d{1,3}>'
-	 * @param  array|string   default values or metadata
+	 * @param  array|string|\Closure  efault values or metadata or callback for NetteModule\MicroPresenter
```
Comment (pavelkouril) [inline src/Application/Routers/Route.php]: You missed a "d" there.

**Comment by richard-ejem** [comment]
fixed and squashed.

**Comment by dg** [comment]
thx

**Pull request merged**